Models/IBusinessLogic.cs

```diff
@@ -32,6 +32,9 @@ public interface IBusinessLogic : INotifyPropertyChanged {
     /// <returns>The result and a nullable string showing the message if something went wrong</returns>
     Task<(LogoutResult, string?)> Logout();
 
+    public QuizManager? getEditQuizManager();
+    void SetEditQuestionQuizManager(QuizManager quizManager);
```

Comment:
Like earlier mentioned, we could put that property also in the IBusinessLogic which might simplify it and then just get rid of the methods

Reply:
Thank you for your input! I've deleted the setter and getter method.